packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -133,21 +158,32 @@ function getLoadDocumentsOptions(filename: string) {
   return loadTypedefsConfig
 }
 
-function getPluginConfig() {
+function getPrismaModels() {
   let prismaModels: Record<string, string> = {}
+
+  // Extract the models from the prisma client and use those to
+  // set up internal redirects for the return values in resolvers.
+  const localPrisma = require('@prisma/client')
+  prismaModels = localPrisma.ModelName
+
+  // This isn't really something you'd put in the GraphQL API, so
+  // we can skip the model.
+  if (prismaModels.RW_DataMigration) {
+    delete prismaModels.RW_DataMigration
+  }
+
+  return prismaModels
+}
+
+function getPluginConfig(side: CodegenSide) {
+  const prismaModels: Record<string, string> = getPrismaModels()
   try {
-    // Extract the models from the prisma client and use those to
-    // set up internal redirects for the return values in resolvers.
-    const localPrisma = require('@prisma/client')
-    prismaModels = localPrisma.ModelName
     Object.keys(prismaModels).forEach((key) => {
-      prismaModels[key] = `@prisma/client#${key} as Prisma${key}`
+      // Post = `@prisma/client#Post as PrismaPost`
```

Comment:
This comment might be a little confusing if you don't know the full history here. I suggest either removing it, or expanding it with more context